### DISCUSSION TITLE
[Feature Request] chained/nested/linked variables

### DISCUSSION THREAD

**Discussion by mikempx** [grafana/grafana-infinity-datasource#696] [Ideas]
My global telco customer is trying to pass data from one variable to another using the Infinity datasource.
example:
Variable _Territory_ with values of North, South, East, and West would drive the content for variable _Market_ .  _Market_ has values like Great Lakes and would only appear if the North Territory were chosen.   Then, if _Market_ Great Lakes is chosen, variable _Region_ would show a list of Great Lake states like MN, OH, IL, IN, WI, PA (or whatever they consider great lakes) only.

**Comment by yesoreyeram** [comment]
That should be already doable. What your data look like? What parser you are using? Provide more details of what you tried.

**Reply by yesoreyeram** [reply]
Example #435

**Reply by yesoreyeram** [reply]
another example: https://play.grafana.org/d/c8416e7a-7cf3-48a1-b714-4a31ce7d0f5c/nested-variables-chained-variables?orgId=1
